components/ledger/internal/adapters/postgres/balance/balance.postgresql.go

```
@@ -1459,100 +1412,97 @@ func (r *BalancePostgreSQLRepository) SyncBatch(ctx context.Context, organizatio
+	for i, balance := range deduped {
+		valuesClauses[i] = fmt.Sprintf("($%d::uuid, $%d::numeric, $%d::numeric, $%d::bigint)",
+			paramIdx, paramIdx+1, paramIdx+2, paramIdx+3)
+		args = append(args, ids[i], balance.Available, balance.OnHold, balance.Version)
+		paramIdx += 4
+	}
+
+	// Shared parameters: updated_at, organization_id, ledger_id
+	nowIdx := paramIdx
+	orgIdx := paramIdx + 1
+	ledgerIdx := paramIdx + 2
+	args = append(args, now, organizationID, ledgerID)
```

golangci-lint (2.11.4)

[error] missing whitespace above this line (no shared variables above append)

(wsl_v5)